static/compatibilities/cert-manager.yaml

```diff
@@ -3,80 +3,17 @@ git_url: https://github.com/cert-manager/cert-manager
 release_url: https://github.com/cert-manager/cert-manager/releases/tag/v{vsn}
 helm_repository_url: https://charts.jetstack.io
 versions:
-- version: 1.16.0
-  kube: ['1.31', '1.30', '1.29', '1.28', '1.27', '1.26', '1.25']
+- version: 1.17.0
+  kube: [ '1.31', '1.30', '1.29', '1.28', '1.27' ]
   requirements: []
   incompatibilities: []
-  chart_version: 1.16.0
 - version: 1.15.0
-  kube: ['1.30', '1.29', '1.28', '1.27', '1.26', '1.25', '1.24']
+  kube: [ '1.31', '1.30', '1.29', '1.28', '1.27', '1.26', '1.25' ]
   requirements: []
   incompatibilities: []
   chart_version: 1.15.0
-- version: 1.13.0
-  kube: ['1.27', '1.26', '1.25', '1.24', '1.23', '1.22', '1.21']
-  requirements: []
-  incompatibilities: []
-  chart_version: 1.13.0
 - version: 1.12.0
-  kube: ['1.30', '1.29', '1.28', '1.27', '1.26', '1.25', '1.24', '1.23', '1.22']
+  kube: [ '1.31', '1.30', '1.29', '1.28', '1.27', '1.26', '1.25', '1.24', '1.23', '1.22' ]
   requirements: []
   incompatibilities: []
   chart_version: 1.12.0
-- version: 1.11.0
```

Comment:
we shouldn't be removing all these previous versions either

Reply:
I removed them while testing locally, didn't mean to commit that